gorules/rules_test.go

```diff
@@ -0,0 +1,19 @@
+package gorules
+
+import (
+	"testing"
+
+	"github.com/quasilyte/go-ruleguard/analyzer"
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestRules(t *testing.T) {
+	testdata := analysistest.TestData()
```

Comment:
Can we include parallel execution?
```suggestion
func TestRules(t *testing.T) {
        t.Parallel()
	testdata := analysistest.TestData()
```

Reply:
Sure: https://github.com/0xPolygon/polygon-edge/pull/1271/commits/5798203c6b32c0899999e92d84fd0bddb3940899